app/src/androidMain/kotlin/de/westnordost/streetcomplete/ui/common/image_select/ImageList.kt

```diff
@@ -0,0 +1,131 @@
+package de.westnordost.streetcomplete.ui.common.image_select
+
+import android.content.res.Configuration
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.Layout
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import de.westnordost.streetcomplete.quests.bicycle_repair_station.BicycleRepairStationService
+import de.westnordost.streetcomplete.quests.bicycle_repair_station.asItem
+import kotlin.math.max
+
+@Composable
+fun <T> ImageList(
+    imageItems: List<ImageListItem<T>>,
+    onClick: (index: Int, item: ImageListItem<T>) -> Unit,
+    modifier: Modifier = Modifier,
+    itemsPerRow: Int = 4,
+    itemRole: Role = Role.Checkbox,
```

Comment:
Hmm, this is a parameter, yet, the composable doesn't seem to do anything with this information. Just passes it back to the caller on click. (But the caller already knows each item's role, otherwise, it wouldn't wouldn't be able to pass it here).